Missing babel-runtime dependency?

It seems that `styled-system` requires `babel-runtime`, but it is not listed as a dependency in `package.json`. When I try to run an example without it, I get:
```
Error: Cannot find module 'babel-runtime/core-js/object/keys'
    at styled-system/dist/index.js:8:13
```

--- a/.babelrc
+++ b/.babelrc
@@ -3,8 +3,5 @@
     "env",
     "stage-0",
     "react"
-  ],
-  "plugins": [
-    "transform-runtime"
   ]
 }

--- a/clean-element/.babelrc
+++ b/clean-element/.babelrc
@@ -3,8 +3,5 @@
     "env",
     "stage-0",
     "react"
-  ],
-  "plugins": [
-    "transform-runtime"
   ]
 }

--- a/clean-element/package.json
+++ b/clean-element/package.json
@@ -21,7 +21,6 @@
     "@compositor/x0": "^3.1.1",
     "ava": "^0.25.0",
     "babel-cli": "^6.24.1",
-    "babel-plugin-transform-runtime": "^6.23.0",
     "styled-components": "^3.1.6"
   },
   "ava": {

--- a/package.json
+++ b/package.json
@@ -28,7 +28,6 @@
     "is-ci": "^1.1.0",
     "ava": "^0.25.0",
     "babel-cli": "^6.24.1",
-    "babel-plugin-transform-runtime": "^6.23.0",
     "babel-preset-env": "^1.5.2",
     "babel-preset-react": "^6.24.1",
     "babel-preset-stage-0": "^6.24.1",

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "styled-system",
-  "version": "2.2.4",
+  "version": "2.2.5-0",
   "description": "Design system utilities for styled-components, glamorous, and other css-in-js libraries",
   "main": "dist/index.js",
   "scripts": {

